dashboard/app/models/activity_section.rb

```diff
@@ -86,6 +87,23 @@ def update_script_levels(script_levels_data)
     end
   end
 
+  # Used for seeding from JSON. Returns the full set of information needed to uniquely identify this object.
```

Comment:
would this be clearer @bethanyaconnor ?
```suggestion
# Used for seeding from JSON. Returns the full set of information needed to uniquely identify this object as well as any other objects it refers to by id. 
```
double checking ahead of time since this comment is duplicated in many places.

Reply:
Yes, I think so. I think "uniquely identify this object" made me think that it was in the scope of only this object, and sometimes you need to know a relationship to uniquely identify it. 

small nit: I'm not sure `id` is right here? You use `key` in this particular function for instance. Can the comment be simplified to "as well as a unique identifier for any other objects it refers to" or something like that?

Reply:
please do nit! I struggled with how to word this, because I wanted something that gave the right idea but is also technically accurate and not too long.

conceptually, you need to be able to be able to identify the object, and put it into the right place in the hierarchy. technically, you are filling all the fields named `id` or `*-id`. Saying "refers to" gives the right idea, but is a bit ambiguous because it makes me think of associations in both directions.

What do you think about **"as well as a unique identifier for any other objects it belongs to"**? That has both the right idea (objects belong to their parents) and implies a technically accurate description (`belongs_to` implies that this object has a field which refers to another object by id).

Reply:
Yeah I like that!